frontend/src/components/interests/tag/Tag.jsx

```diff
@@ -11,10 +11,10 @@ export default function Tag({ label }) {
   useEffect(() => {
     // When the internal state of the tag is changed, update the context state.
     if (selectState) {
-      addToSelectedTags(label)
+      addToSelectedTags(`#${label}`)
       return
     }
-    removeFromSelectedTags(label)
+    removeFromSelectedTags(`${label}`)
```

Comment:
bit confused by one has a # and the other one doesn't

Reply:
That was a mistake, fixed in the latest commit. Thanks for picking that up!